src/pages/Rockets.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 // import Rocket from '../components/Rocket';
-import Rocket from '../components/Rocket';
+// import Rocket from '../components/Rocket';
+import Rockets from '../components/Rocket';
 
-const Rockets = () => (
+const RocketsPage = () => (
   <>
```

Comment:
Adding Fragments <> is not necessary here because there are no adjacent sibling elements. [More about React fragments](https://www.javatpoint.com/react-fragments)